packages/math-input/src/components/keypad-container.tsx

```diff
@@ -89,7 +95,7 @@ class KeypadContainer extends React.Component {
         // Throttle the resize callbacks.
         // https://developer.mozilla.org/en-US/docs/Web/Events/resize
         if (this._resizeTimeout == null) {
-            this._resizeTimeout = setTimeout(() => {
+            this._resizeTimeout = window.setTimeout(() => {
```

Comment:
`setTimeout` is ambiguous and TS didn't know what the return value was. By setting it as `window.setTimeout` we were able to determine a return type. Is this a problem for SSR?

Reply:
Still threw an error. Are we concerned by this? I can look into it more if so.

Reply:
Nah. Let's move on. :)